src_cpp/opennnBridge/nerlWorkerOpenNN.cpp

```diff
@@ -28,7 +30,7 @@ namespace nerlnet
     void NerlWorkerOpenNN::perform_training()
     {
         this->_training_strategy_ptr->set_data_set_pointer(this->_data_set.get());
-
+        this->_training_strategy_ptr->get_loss_index_pointer()->set_regularization_method(LossIndex::RegularizationMethod::L2); // ! ADDED NOW
```

Comment:
No need. Please validate that this is rebased on top of master.
In master there is already implementation for regularization.

https://github.com/leondavi/NErlNet/blob/c01ac0b39a8df383cea8fdf0b857f164780e8418/src_cpp/opennnBridge/nerlWorkerOpenNN.cpp#L175